Blish HUD/GameServices/ArcDps/SocketListener/RawCombatEventArgs.cs

```diff
@@ -7,7 +7,13 @@ public class RawCombatEventArgs : EventArgs {
 
         public enum CombatEventType {
 
+            /// <summary>
+            /// ArcDps calculations and display values.
+            /// </summary>
             Area,
+            /// <summary>
+            /// Local recording of the player character bound to the signed-in account of this GW2 instance similar to the in-game combat log in the chat window.
+            /// </summary>
```

Comment:
Me being picky again.
I'd say it'd be better to distinguish the difference between the two better: Area is calculated and shows displayed estimates, Local is an exact recording only available for the player's character.